core/vibes/soul/sections/footer/index.tsx

```diff
@@ -39,17 +39,21 @@ interface Props {
   className?: string;
 }
 
-export const Footer = function Footer({
-  logo,
-  sections,
-  contactInformation,
-  paymentIcons,
-  socialMediaLinks,
-  copyright,
-  className = '',
-}: Props) {
+export const Footer = forwardRef(function Footer(
+  {
+    logo,
+    sections,
+    contactInformation,
+    paymentIcons,
+    socialMediaLinks,
+    copyright,
+    className = '',
+  }: Props,
+  ref: Ref<HTMLDivElement>,
```

Comment:
Add a ref to the VIBES footer, corresponding PR here: https://github.com/makeswift/vibes/pull/285